components/UptimeMonitors/UptimeMonitors.test.js

```diff
@@ -40,4 +40,24 @@ describe("UptimeMonitors", () => {
       expect(screen.getByText("Always Down")).toBeInTheDocument();
     });
   });
+
+  test("should render uptime monitors sorted by title", () => {
+    const statusPage = Object.assign({}, statusPageMock);
+
+    build({ statusPage });
+
+    const updates = screen.getAllByTestId("UptimeMonitor");
+
+    expect(updates[0].querySelector("h2").innerHTML).toContain(
+      statusPageMock.uptime_monitors[0].title
+    );
+
+    expect(updates[1].querySelector("h2").innerHTML).toContain(
+      statusPageMock.uptime_monitors[1].title
+    );
+
+    expect(updates[2].querySelector("h2").innerHTML).toContain(
+      statusPageMock.uptime_monitors[2].title
+    );
+  });
```

Comment:
I don't think this test is testing anything, as the order is already "correct" in the mock. 
You should probably randomise the order first and then assert it's ordered correctly.

Reply:
Ah, nice catch. I have improved this. Before the `statusPage` is built I am now assigning random monitors

```
    statusPage.uptime_monitors = multipleMonitorsMock.uptime_monitors;
    build({ statusPage });
```